src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import moment from "moment";
+import React from "react";
+
+const SearchResults = prob => {
+  return (
+    <table className="table">
+      <thead className="thead-dark">
+        <tr>
+          <th scope="col">id</th>
+          <th>title</th>
+          <th>first name</th>
+          <th>surname</th>
+          <th>email</th>
+          <th>room id</th>
+          <th>check in date</th>
+          <th>check out date</th>
+          <th>nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {prob.results.map((Bookings, index) => {
```

Comment:
There's a few things about the `Bookings` variable naming that I want to comment on:

1. I would switch it to lowercase - remember that this is just Javascript here, so the normal naming convention for variables is to start with a lowercase letter
2. Does this variable represent a single booking or multiple bookings? How could we improve our naming to be clear about which of those it is?